### PR TITLE
Feat(variables-scss): Add debug option for exporter #DS-1505

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Adding option to toggle generating original unformatted data from Supernova to our exporter.

You can enable this feature by setting the `generateOriginalDataFiles` option in the `config.local.json` file. 
By default, this option is set to `false`.

### Additional context

> [!NOTE]
> The `generateDisclaimer` option has been removed from the config for clarity. 
> We never used this option. Our disclaimer is always displayed by default.

### Issue reference

[Exporter | add debug option for exporter](https://jira.almacareer.tech/browse/DS-1505)

### CODE REVIEW

File: exporters/variables-scss/src/index.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
 import { FileHelper } from '@supernovaio/export-helpers';
 import {
   AnyOutputFile,
+  OutputTextFile,
```
Comment (curdaj) [inline exporters/variables-scss/src/index.ts:4]: 😄👍

**Review by curdaj — APPROVED** [review]

File: exporters/variables-scss/src/index.ts [hunk 2]
```diff
@@ -46,35 +48,34 @@ Pulsar.export(async (sdk: Supernova, context: PulsarContext): Promise<Array<AnyO
...
-    let cache: string[] | null = [];
-    const str = JSON.stringify(
-      obj,
-      (key, value) => {
-        if (typeof value === 'object' && value !== null) {
-          if (cache?.includes(value)) {
-            return 'CIRCULAR_REFERENCE';
+  // Export the original data only if 'generateOriginalDataFiles' is set to true in config.local.json
+  if (exportConfiguration.generateOriginalDataFiles) {
+    const safeStringify = (obj: object) => {
```
Comment (literat) [inline exporters/variables-scss/src/index.ts:53]: IMHO, I would suggest moving this function to something like an `utils.ts/helpers.ts` and creating a test for it :-)

**Review by literat — APPROVED** [review]
LGTM 👍

File: exporters/variables-scss/src/index.ts (hunk 2 above)
Comment (pavelklibani) [reply exporters/variables-scss/src/index.ts]: good point - moved function to separate file and created test for it

**Comment by pavelklibani** [comment]
rebased to main

**Pull request merged**